css/styles.css

```diff
@@ -62,7 +62,7 @@ body {
     height: 100vh;
     margin: 0;
     padding: 0;
-    font-family: "MGS2", sans-serif;
```

Comment:
I'm not interested in changing the font to Lato. It makes fast ui look wrong, and is also rather arbitrary. Same goes for the snow bg.